File: 백준/Bronze/14470. 전자레인지/전자레인지.py
a = int(input())
b = int(input())
c = int(input())
d = int(input())
e = int(input())

cnt = 0

if a < 0:
    cnt += d

while a < 0:
    a += 1
    cnt += c

while a < b:
    a += 1
    cnt += e

print(cnt)

File: 백준/Bronze/14470. 전자레인지/전자레인지.cs
int a = int.Parse(Console.ReadLine());
int b = int.Parse(Console.ReadLine());
int c = int.Parse(Console.ReadLine());
int d = int.Parse(Console.ReadLine());
int e = int.Parse(Console.ReadLine());

int cnt = 0;

if (a < 0)
    cnt += d;

while (a < 0)
{
    a += 1;
    cnt += c;
}

while (a < b)
{
    a += 1;
    cnt += e;
}

Console.WriteLine(cnt);